### ISSUE TITLE
Dynamic service fails while stopping (uploading workspace)

### ISSUE THREAD

**Issue by elisabettai** [ITISFoundation/osparc-issues#1143]
**Long Story Short**
On osparc.io, I got a failure while stopping a jupyter-ml-tensorflow service (node uuid: `a9c844c8-4245-48e0-9231-db0bcf28f1fc`) 

**Error**
In the dynamic sidecar, I found this:

```
log_level=ERROR | log_timestamp=2023-10-02 08:52:57,115 | log_source=simcore_sdk.node_ports_common.filemanager:upload_path(327) | log_uid=None | log_msg=The upload failed with an unexpected error:

Traceback (most recent call last):

  File "/home/scu/.venv/lib/python3.10/site-packages/simcore_sdk/node_ports_common/filemanager.py", line 312, in upload_path

    store_id, e_tag, upload_links = await _upload_to_s3(

  File "/home/scu/.venv/lib/python3.10/site-packages/simcore_sdk/node_ports_common/filemanager.py", line 380, in _upload_to_s3

    await r_clone.sync_local_to_s3(

  File "/home/scu/.venv/lib/python3.10/site-packages/simcore_sdk/node_ports_common/r_clone.py", line 222, in sync_local_to_s3

    await _sync_sources(

  File "/home/scu/.venv/lib/python3.10/site-packages/simcore_sdk/node_ports_common/r_clone.py", line 192, in _sync_sources

    await _async_r_clone_command(

  File "/home/scu/.venv/lib/python3.10/site-packages/simcore_sdk/node_ports_common/r_clone.py", line 104, in _async_r_clone_command

    raise RCloneFailedError(

simcore_sdk.node_ports_common.r_clone.RCloneFailedError: Command rclone --config /tmp/tmp3bcuq_o2 --retries 3 --transfers 5 --use-mmap --buffer-size 0M --stats-log-level NOTICE --stats 0.5s sync /dy-volumes/home/jovyan/work/workspace s3-destination:production-simcore/7e2883ce-5ed6-11ee-9f5c-02420a0bfeea/a9c844c8-4245-48e0-9231-db0bcf28f1fc/workspace --progress --links --verbose finished with exit code=1:

2023-10-02 08:52:27 ERROR : workspace: failed to get info about directory entry "/dy-volumes/home/jovyan/work/workspace/workspace/__pycache__": lstat /dy-volumes/home/jovyan/work/workspace/workspace/__pycache__: permission denied

2023-10-02 08:52:27 ERROR : workspace: failed to get info about directory entry "/dy-volumes/home/jovyan/work/workspace/workspace/__pycache__": lstat /dy-volumes/home/jovyan/work/workspace/workspace/__pycache__: permission denied
```
**Additional context**
There are many files in the workspace folder and some were generated by decompressing an archive with `tar -xzvf`
And here is the full log: 
[_dy-sidecar_a9c844c8-4245-48e0-9231-db0bcf28f1fc.1.6ju23jtns0dbqeyzh3kh2uu3u_logs.txt](https://github.com/ITISFoundation/osparc-issues/files/12781329/_dy-sidecar_a9c844c8-4245-48e0-9231-db0bcf28f1fc.1.6ju23jtns0dbqeyzh3kh2uu3u_logs.txt)

**Comment by GitHK** [comment]
As an update, this is a permissions issue and is unrelated to rclone or it's functionality.
The user under which the rclone process is ran has no permissions and this breaks.
This happened because the data ws imported via tar files which preserve the original permissions of the files.

**Comment by elisabettai** [comment]
**Conclusion**: we should warn somewhere (in a FAQ) user to don't use tar, but zip, as per comment above.

To recover the data this is what @GitHK did:

- ssh into the sidecar in protainer
- install zip via apt
- cd in the right path inside /dy-volumes which is what emulates the local fs in the user serivce : /dy-volumes/home/jovyan/work/workspace
- zip the contents of the fodler zip -r /tmp/arhvied_worksapce.zip  .
- remove all files and folders listed by ls -lah except (.hidden_do_not_remove). I manually added them, there were only ~10 files.
- opnend director-v2 and saved the node as usual (it closed succesfully)

Closing since user is happy: data were recovered and learned something new about tarring/zipping and permissions. 😉
